aten/src/ATen/native/Pow.cpp

```diff
@@ -20,7 +20,7 @@ Tensor& pow_out(Tensor& result, const Tensor& base, const Tensor& exp) {
 Tensor& pow_out(Tensor& result, const Tensor& base, Scalar exp) {
   // Numpy compatibility check:
   TORCH_CHECK(!(isIntegralType(base.scalar_type()) &&
-              exp.isIntegral() && exp.toLong() < 0),
+              exp.isIntegral(false) && exp.toLong() < 0),
```

Comment:
why does one of these isIntegral* calls pass a flag and the other doesn't?

Reply:
fixed. they should have both been true not false, since really the point of this assertion is to avoid negative exponents on non-floating types.